gemrb/plugins/AREImporter/AREImporter.cpp

```diff
@@ -601,11 +601,6 @@ Map* AREImporter::GetMap(const char *ResRef, bool day_or_night)
 		ip->StrRef = StrRef; //we need this when saving area
 		ip->SetMap(map);
 		ip->Flags = Flags;
-		// ensure repeating traps are armed, fix for bg2 ar1404 mirror trap to fire
-		// BUT probably not in PST, since that breaks the mausoleum portal in ar0200
-		if (ip->TrapResets() && !core->HasFeature(GF_PST_STATE_FLAGS)) {
-			ip->Trapped = true;
-		}
```

Comment:
that's the main point of resetting traps, they reset. However Zelazko put it better, was this the real problem you faced?

> resettable traps can't be disarmed after they reappear

Reply:
Resetting traps are supposed to reset _after triggering_, not after being disarmed!  (Even if they were to reset, it wouldn't be triggered by reload but rather some in-game event.)  After a trap is properly disarmed by a thief, it shouldn't ever reappear on it's own and yes I've tested this.  You _can_ disarm them again, but it's both a chore and an XP farm.

Reply:
Had amazing luck with this today, just because I was using a different ie data tool.